Resources/Prototypes/Polymorphs/polymorph.yml

```diff
@@ -66,6 +66,57 @@
   transferHumanoidAppearance: true
   inventory: Transfer
 
+- type: polymorph
+  id: HumanMutationMorph
+  entity: MobHuman
+  forced: true
+  transferName: true
+  # I would like to use transferHumanoidAppearance, but it doesn't work the way you'd expect yet
```

Comment:
be more specific, whats wrong with it

Reply:
The issue with transferHumanoidAppearance is that it results in no appearance changes whatsoever on transformation. When you transform from a slime person, for example, to any other species, you still keep the characteristic transparency of a slime person. If you transform from a lizard to a human, you keep the tail. The behaviour I would expect is that whatever can be maintained within the constraints (so like hair color) gets carried over, and anything that violates the constraints is shuffled/randomized. I can modify the comment, if you like, but I don't know how useful that is.